modules/test/playwright/tests/frontend-data-set-views-web/itemActions.spec.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {actionsPageTest} from './fixtures/actionsPageTest';
+import {dataSetsPageTest} from './fixtures/dataSetsPageTest';
+import {viewsPageTest} from './fixtures/viewsPageTest';
+
+export const test = mergeTests(
+	actionsPageTest,
+	dataSetsPageTest,
+	featureFlagsTest({
+		'LPS-164563': true,
+		'LPS-194395': true,
+	}),
+	loginTest,
+	viewsPageTest
+);
+
+const LINK_ITEM_ACTION_NAME = 'Link item action';
+
+test('Create a Link Item Action', async ({
+	actionsPage,
+	dataSetsPage,
+	page,
+	viewsPage,
+}) => {
+	await test.step('Create Data Set', async () => {
+		await dataSetsPage.goto();
+		await dataSetsPage.createSampleDataSetUI();
```

Comment:
[not in scope of PR, but...] `UI` suffix in `createSampleDataSetUI` seems odd to me, we are not creating UI. I guess thinking here was that this creates a sample _through_ UI, as opposed to API. Can this be just `createSampleDataSet()`? If we ever make an API util, it can be something like `generateSampleDataSet()`.

Reply:
I like that proposal Marko. This is a leftover from the times where we considered creating things via API vs via UI.

Now debate is whether doing so using a OSGi module or using API calls, or a mix of the two depending on the case.

But the core business of tests is to interact with the UI so that shall be taken as granted